src/App.js

```diff
@@ -1,80 +1,18 @@
 import React from 'react';
 import './App.css';
-// Move each BEM block to a separate component (file) and import them here
+import Header from './components/Header/Header';
+import './components/Header/Header.css';
+import Welcome from './components/Welcome/Welcome';
+import './components/Welcome/Welcome.css';
+import Article from './components/Article/Article';
+import './components/Article/Article.css';
```

Comment:
```suggestion
import Welcome from './components/Welcome/Welcome';
import Article from './components/Article/Article';
```
import css here not necessary, it already importet in jsx files